notebooks/BTA_array_producer.ipynb

```diff
@@ -0,0 +1,2322 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #26.    istau = genlep_pa1G.pdgId == 15
</pre><p>shouldn't it be <code>istau = abs(genlep_pa1G.pdgId) == 15</code>? there are tau- (ID 15) and tau+ (ID -15)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cms-btv-pog/BTVNanoCommissioning/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>@milee please fix the tau bug in the notebook as well</p>